examples/soak/src/crux/soak/main.clj

```diff
@@ -154,10 +156,15 @@
                             (map first))]
     (resp/response (render-homepage location-names))))
 
+(defn status-handler [req {:keys [crux-node]}]
+  (let [status (api/status crux-node)]
```

Comment:
anything we can check in this status response to check everything's a-ok?

Reply:
Oh whoops I meant to add a check completely slipped my mind - I was thinking of more less taking it straight from http-server, which checks for `:crux.zk/zk-active?`? (Out of interest, what is that, exactly?)